src/ruby/ext/grpc/extconf.rb

```diff
@@ -65,13 +66,14 @@
 output_dir = File.expand_path(RbConfig::CONFIG['topdir'])
 grpc_lib_dir = File.join(output_dir, 'libs', grpc_config)
 ENV['BUILDDIR'] = output_dir
+make_flags = use_embedded_ssl ? 'HAS_SYSTEM_OPENSSL_ALPN=true HAS_EMBEDDED_OPENSSL_ALPN=false' : ''
 
 unless windows
   puts 'Building internal gRPC into ' + grpc_lib_dir
   nproc = 4
   nproc = Etc.nprocessors * 2 if Etc.respond_to? :nprocessors
   make = bsd ? 'gmake' : 'make'
-  system("#{make} -j#{nproc} -C #{grpc_root} #{grpc_lib_dir}/libgrpc.a CONFIG=#{grpc_config} Q=")
+  system("#{make} -j#{nproc} -C #{grpc_root} #{grpc_lib_dir}/libgrpc.a CONFIG=#{grpc_config} Q= #{make_flags}")
```

Comment:
I think I got this reversed.

Reply:
These flags don't appear to be in `master`. I was testing against gRPC v1.30.2.

Reply:
It looks like https://github.com/grpc/grpc/pull/23957 removed the ability to use the system OpenSSL. 😦